client/src/components/LandingPage/Hero.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import {
+  Row,
+  Col,
+  Image,
+  Button,
+} from 'antd';
+import heroImage from '../../assets/img/hero.png';
+import './LandingPage.css'
+
+function Hero() {
+  return (
+    <div className="hero-sec">
+      <Row gutter={[8, 8]} style={{ marginRight: 0 }}>
```

Comment:
we use inline stile just for test not to merge it into the main so dont push any inline stilyle on Github